sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1680,9 +1680,7 @@ object GpuOverrides extends Logging {
       }),
     expr[KnownFloatingPointNormalized](
```

Comment:
Are there tests in `hash_aggregate_test.py` and `join_test.py` that would pass now if we require `KnownFloatingPointNormalized` to be on the GPU? We have it in a number of places under `allow_non_gpu`.

Reply:
Great point! I am not sure if we want to do this as a part of this PR though. I can push a separate PR after creating a ticket.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/6415

Reply:
I would very much like to see the https://github.com/NVIDIA/spark-rapids/issues/6415 issue handled here, since this PR is allowing more types for `KnownFloatingPointNormalized`. Conversely, it would be nice to punt on the `KnownFloatingPointNormalized` change until the tests can come along with it.

Reply:
To keep the scope of the PR simple, I would like to handle the allowing `KnownFloatingPointNormalized` for all types as a part of #6415. @jlowe are you OK with that?

Reply:
Fine with me as long as it's tracked.